### ISSUE TITLE
Add cookies import feature

### ISSUE THREAD

**Issue by metastable-void** [menhera-org/TabArray#212]
**Is your feature request related to a problem? Please describe.**
Some users are frustrated when they find themselves logged out on websites when they create a container.

**Describe the solution you'd like**
- Allow users to import cookies from another container (including the default container) when they create a container.
- Allow users to import cookies from another container afterwards.

**Describe alternatives you've considered**
Not using containers.

**Additional context**
https://addons.mozilla.org/en-US/firefox/addon/container-tab-groups/reviews/1962335/
> It is closer to Chrome's tab group function than other containers, but you should be aware that adding this extension is not user-friendly in that it asks you to re-login on all pages, and when you delete this extension, all tabs in the container you created disappeared. Also, when you restore the lost tabs, the containers are still colored even after the extension was deleted, so I think you should fix that.
> 
> Translated with www.DeepL.com/Translator (free version)

**Comment by metastable-void** [comment]
In the developer tools, it worked:

```javascript
ccs = gServiceRegistry.getService('CookieCopyService') ;
await ccs.copyAllCookiesToContainer('firefox-container-65', 'firefox-container-475');
```

**Comment by metastable-void** [comment]
Implemented in 6e96245944b6402bcfe04cc8b024f220fc5bea98. (Import cookies when creating a container)
